### DISCUSSION TITLE
Ayuda para Kata 2

### DISCUSSION THREAD

**Discussion by Are31** [LaunchX-InnovaccionVirtual/CursoIntroPython#326] [Q&A] [answered]
No se exactamente que empezara a realizar si me pudieran ayudar por favor

**Comment by ArielCanedo** [comment, marked as answer]
Claro que sí
Primero instala python sino lo tienes desde su página web o en Windows
también se puede desde el store

Luego en cmd puedes verificar que esté Python con
python --version
Ahí te arroja la versión

Para crear el entorno virtual usamos

python -m venv env (dónde env será el nombre del entorno virtual, puedes
poner test u otra cosa)

Después de correr ese comando bien sigue activarlo que es así

env/scripts/activate

Ahí la línea de cmd siguiente debe empezar con el nombre del entorno
virtual para que sea correcto

Luego instalas lo que viene en el documento
Y le tomas capturas a cmd lo puedes guardar en Word luego hacerlo pdf y va
a repositorio de GitHub.

**Reply by Are31** [reply]
gracias, lo único que me falta es lo de instalar un paquete, que es la segunda parte, no entiendo, me podría ayudar por favor?

**Comment by Are31** [comment]
![image](https://user-images.githubusercontent.com/99683479/154871116-f68d6fa0-2f4d-4a97-8f70-4ae17542dd48.png)

**Comment by ArielCanedo** [comment]
Ah okay
El código para instalarlo es

pip install python-dateutil

**Comment by JuanSalvador73** [comment]
Instalas el entorno virtual como se describe, y luego le sacas screenshots y los pegas en un documento.
